src/main/java/homework4/Main.java

```diff
@@ -0,0 +1,16 @@
+package homework4;
+
+import java.util.HashMap;
+
+public class Main {
+    public static void main(String[] args) {
+        Bank<Asset> bank = new Bank<>("Volodymirska 16");
```

Comment:
The main reson of generic - use with some concretic type
example: Bank\<Gold\>. In internal you have (in your case) array[] of type Gold

Reply:
> 
> 
> The main reson of generic - use with some concretic type
> example: Bank<Gold>. In internal you have (in your case) array[] of type Gold

Lead idea was to allow Bank operate with different type of Assets at same time, I just didnt understud task properly